tests/frontend/test_kibana_app/cypress/integration/pageobjects/filters/filters.page.js

```diff
@@ -7,4 +7,5 @@ export const stablishedFilter = '[data-test-subj="filter filter-enabled filter-k
 export const pinFilterAction = '[data-test-subj="pinFilter"]';
 export const selectedOperator = '[title="is"]';
 export const pinnedFilter= '[data-test-subj="filter filter-enabled filter-key-rule.level filter-value-7 filter-pinned"]';
-export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]'
\ No newline at end of file
+export const eventsButton = '//*[contains(@class,"euiTabs")]//*[contains(text(),"Events")]';
```

Comment:
You know I don't like to use XPath selectors, so take this recommendation with a grain of salt.

In this case we could find several selectors that would work and would be easier to read and to understand, and also perform better (in execution time) but, if this is the only way you could make it work, leave it as it is and we could find a better selector in the future.